qiskit/qpy/binary_io/circuits.py

```diff
@@ -320,12 +320,16 @@ def _read_instruction(
         )
         if condition is not None:
             warnings.warn(
-                f"The .condition attribute on {gate_name} will be loaded as an IfElseOp "
-                "starting in Qiskit 2.0",
-                FutureWarning,
+                f"The .condition attribute on {gate_name} can not be "
+                "represented in this version of Qiskit. It will be "
+                "represented as an IfElseOp instead.",
+                UserWarning,
```

Comment:
Maybe this ought to be `QPYLoadingDeprecatedFeatureWarning` from `qiskit.qpy.exceptions`?  Same for the other warning in this file.

Reply:
I looked at the description of the warning class and it felt more like a warning raised on loading a deprecated but not yet removed function. I saw `UserWarning` being used for this kind of warning in other places in QPY so I went with it. If we want a custom warning type here I can add a new type that is specific to this situation.